src/Routes/ImageManager/Images.js

```diff
@@ -81,6 +100,106 @@ const Images = () => {
     }),
     shallowEqual
   );
+
+  useEffect(() => {
+    if (data) {
+      setRows(
+        flatten(
+          data.map(item => {
+            const statusIconHash = {
+              "Ready": <CheckCircleIcon color="var(--pf-global--success-color--100)" />,
+              "Image build in progress": <InProgressIcon color="var(--pf-global--palette--blue-400)" />,
+              "Building error": <ExclamationCircleIcon color="var(--pf-global--danger-color--100)" />
+            } 
+            return [
+              {
+                id: item.id,
+                currentStatus: item?.request.status,
+                rowArray: [
+                  item.request.name,
+                  item.request.version,
```

Comment:
`item.Version`

Reply:
BTW, why did you add `rowArray`?

Reply:
I used `rowArray` in order to get sorting to work but I will revert it back as I'm going to remove the sorting functionality.